.github/workflows/ci.yml

```diff
@@ -90,3 +88,13 @@ jobs:
         run: docker build -t ubuntucontainer tss-esapi/tests/ --file tss-esapi/tests/Dockerfile-ubuntu
       - name: Run the tests
         run: docker run -v $(pwd):/tmp/rust-tss-esapi -w /tmp/rust-tss-esapi/tss-esapi ubuntucontainer /tmp/rust-tss-esapi/tss-esapi/tests/valgrind.sh
+  # Check that the documentation builds as well.
```

Comment:
Could you insert a newline above this line so that there's some breathing space? :)